src/wifi/mod.rs

```diff
@@ -1,43 +1,149 @@
+use manager::NetworkManager;
+use device::Device;
+use device;
+
+
+#[derive(Debug)]
+pub struct AccessPoint {
+    pub path: String,
```

Comment:
Should this be a dbus::Path instead?
https://docs.rs/dbus/0.5.2/dbus/struct.Path.html